CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🎁 Starting with this release, installing VAST on any Linux becomes
+  significantly easier: A static binary will be provided with each release on
+  the GitHub releases page. The direct link to the latest version is
+  https://github.com/tenzir/vast/releases/latest/download/vast-linux-static.tar.gz
```

Comment:
This this link may change because of a variety of reasons, I wouldn't hardcode in the changelog. In a year from now, I bet things will be different...but the changelog won't be.

Reply:
Fair enough.

Reply:
I don't agree. The changelog should reflect our current set of changes, and this is a user-facing change.

If the link changes again in the future, we will add another changelog entry.

Reply:
I think we can add the link to the docs and the release notes, maybe it even qualifies for the blog post, that way it does not necessarily have to be in the changelog entry.